macros/Cargo.toml

```diff
@@ -12,6 +12,10 @@ version = "0.1.1"
 [lib]
 proc-macro = true
 
+[features]
+# WARNING: for internal use only, not covered by semver guarantees
+unstable-test = []
```

Comment:
~~I don't think the feature is necessary in this crate. The proc macro code is *emitting* `cfg(feature = "unstable-test")` but not using the Cargo feature.~~

EDIT: after reading the rest of the PR, I see that we do want this